aws/credentials/ssocreds/testdata/custom_cached_token.json

```diff
@@ -0,0 +1,4 @@
+{
+  "accessToken": "ZhbHVldGhpcyBpcyBub3QgYSByZWFsIH",
+  "expiresAt": "2021-01-19T23:00:00Z"
```

Comment:
this is the value im referring to. when i run this through the debugger the unmarshaler (in `loadTokenFile`) seems to be interpreting this way in the future. am i missing something here?

Reply:
see above thread https://github.com/aws/aws-sdk-go/pull/4885#discussion_r1253602360